test/cluster/clusterInfo.spec.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-env jest */
+
+const { cluster } = require('../../lib');
+const { ConnectionFactory } = require('../setup-database');
+
+describe('cluster.info()', () => {
+  it('should retrieve a JS object containing cluster information', () => {
+    const coordinatorConnection = ConnectionFactory(5821);
+
+    return cluster.info(coordinatorConnection).then(res => {
+      const { nodes } = res.body;
+      const coordinator = nodes.find(node => node.search(/5821/) > 0);
+      const participant = nodes.find(node => node.search(/5822/) > 0);
```

Comment:
Nitpick-ish: Instead of `node.search(regex) > 0`, you can just more directly do `node.includes(number)` here to accomplish the same purpose a little more clearly.